hocr-check

```diff
@@ -83,7 +83,8 @@ def mostly_nonoverlapping(boxes, significant_overlap=0.2):
 ################################################################
 
 parser = argparse.ArgumentParser(
-    description="Check the given file for conformance with the hOCR format spec"
+    description=("Check the given file for conformance with the hOCR"
+                 "format spec")
```

Comment:
Here the blank got lost ("hOCRformat").

Reply:
Okay, do you want to fix these missing spaces?